<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_serial_port">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>334</width>
         <height>301</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Select Serial Port</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="versionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Version</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="serialNumberLabel">
          <property name="text">
           <string>Serial Number:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="locationLabel">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="pidLabel">
          <property name="text">
           <string>Product ID:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="manufacturerLabel">
          <property name="text">
           <string>Manufacturer:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="serialPortInfoListBox"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="vidLabel">
          <property name="text">
           <string>Vendor ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_digit_port_only">
          <property name="text">
           <string>Show digitizer port only</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="checkBox_measure_position_save_on_exit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>334</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Measure window position</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_main_position_save_on_exit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>334</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Main window position</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>205</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,0">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Arm 1 Lenght</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_arm1_length">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_arm1_units">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,0">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Arm 2 Lenght</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_arm2_length">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_arm2_units">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="10" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="step_label">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="auto_step">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_step_units">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="11" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="language">
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ComboBox_language">
            <property name="font">
             <font>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="currentText">
             <string>English</string>
            </property>
            <item>
             <property name="text">
              <string>English</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>German</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Czech</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,0">
          <item>
           <widget class="QPushButton" name="choose_arms_color">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>choose color pen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="arms_color">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Units</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="unit_select">
            <item>
             <property name="text">
              <string>mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>inch</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="ShortCuts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Shortcuts</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>470</y>
         <width>411</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,2,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>export dir  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_dxf_dir">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_browse_dxf">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>security</string>
      </attribute>
      <widget class="QComboBox" name="comboBox_datasource">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Serial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modbus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Emulate</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="Serial_source_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Data source</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
   <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </item>
  <item row="3" column="1">
   <widget class="QPushButton" name="exportJsonButton">
    <property name="text">
     <string>Export JSON</string>
    </property>
   </widget>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
